platform-configs/include/msi-z690-common.robot

```diff
@@ -128,3 +131,4 @@ Power On
     # read the old output
     Telnet.Read
     Rte Power On
+    Power Cycle On
```

Comment:
Is this the correct order? I'd expect it to be before `Rte Power On`. You first have to plug the power cord, and then press the button.

Reply:
This was fixed in https://github.com/Dasharo/open-source-firmware-validation/commits/regression-msi/